flask_rebar/request_utils.py

```diff
@@ -10,6 +10,7 @@
 from __future__ import unicode_literals
 
 import copy
+from collections import Mapping
```

Comment:
On Python 3.7+ this will generate a deprecation warning. See https://github.com/jab/bidict/blob/master/bidict/compat.py#L41 
One more thing to move into the compat module.

Reply:
Haha gdammit.